cypress/utils/helpers.js

```diff
@@ -1,10 +1,9 @@
-import Chance from 'chance';
-
-const chance = new Chance();
+export function pickTargetCategory(productsList) {
+  return Cypress._.sample(productsList.collections);
+}
 
-export function pickRandomProduct(listOfProducts) {
-  const selectedCategory = chance.pickone(listOfProducts.collections);
-  const selectedItem = chance.pickone(selectedCategory.items);
-  selectedItem.linkUrl = selectedCategory.linkUrl;
-  return selectedItem;
+export function pickTargetProducts(category, quantity = 1, isShopPage = false) {
+  const items = isShopPage ? Cypress._.take(Cypress._.values(category.items), 4)
+    : Cypress._.values(category.items);
```

Comment:
prefer this way of doing ternary if it looks complex
```
  const items = (isShopPage)
        ? Cypress._.take(Cypress._.values(category.items), 4)
        : Cypress._.values(category.items);
```

Reply:
in the case of simple ternary use one-liner
`const smth = (condition) ? 5 : 6;`

Reply:
done